server/utils/applicationUtils.ts

```diff
@@ -13,6 +13,17 @@ export const dashboardTableRows = (applications: Array<Application>): Array<Tabl
     ]
   })
 }
+
+export const submittedApplicationTableRows = (applications: Array<Application>): Array<TableRow> => {
```

Comment:
would it be worth having these as one function that takes in a status as an argument and returns the desired date based on that status?